Stackdriver-Webhook/index.js

```diff
@@ -57,6 +56,59 @@ async function postToGoogleMonitoring(response) {
 		timeSeriesData[i] = parseTimeSeriesData(metric, dataPoint, testId, nodeName);
 	}
 
+	/** If Test type is Trace route then compute RTT, Packet loss, #Hops */
+	if (response.TestDetail.TypeId == 12) {
+	
+		let sumPingTime = 0;
+		let packetLossCounter = 0;
+		let pingCounter = 0;
+
+		let numberOfHops = response.Diagnostic.TraceRoute.Hops.length
+
+		for (var i = 0; i < 3; i++) {
+			if (response.Diagnostic.TraceRoute.Hops[numberOfHops - 1].RoundTripTimes[i] != null) {
+
+				sumPingTime += response.Diagnostic.TraceRoute.Hops[numberOfHops - 1].RoundTripTimes[i];
+				pingCounter++;
+			}
+			if (response.Diagnostic.TraceRoute.Hops[numberOfHops - 1].RoundTripTimes[i] == null) {
+				packetLossCounter++;
+			}
+		}
+
+		let rtt = (sumPingTime / pingCounter).toFixed();
+		let packetloss = (33.333 * packetLossCounter).toFixed();
```

Comment:
Webhook does not give **packet-loss percentage**.
I had to write logic to calculate this metric.

we should have 3 pings at every hop.
So 33.333 was derived by dividing 100/ total number of hops = 33.333

For example, If there are 2 pings failing, Then packet loss would be
33.333 * 2 = 66.666

If all 3 ping fails it would be 
33.333 * 3 = 100